src/mame/jpm/jpmsys5.h

```diff
@@ -113,6 +113,8 @@ class jpmsys5_state : public driver_device
 
 	uint16_t unk_r(offs_t offset, uint16_t mem_mask = ~0);
 
+	static constexpr uint8_t reel_interface_table[4] = {0x0a, 0x09, 0x06, 0x05};//An interface maps the 2-bit values to meaningful Starpoint patterns (Proconn do this too, put in steppers?)
+
```

Comment:
You need to either make this `inline` as well as `static` and `constexpr` or instantiate it somewhere, otherwise you can get undefined reference errors.